build/Common.props

```diff
@@ -19,12 +19,13 @@
       Please sort alphabetically.
       Refer to https://docs.microsoft.com/nuget/concepts/package-versioning for semver syntax.
     -->
-    <MicrosoftExtensionsLoggerVer>[2.0,)</MicrosoftExtensionsLoggerVer>
+    <MicrosoftExtensionsLoggerVer>[8.0.0,)</MicrosoftExtensionsLoggerVer>
     <MicrosoftSourceLinkGitHubPkgVer>[1.0.0,2.0)</MicrosoftSourceLinkGitHubPkgVer>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="System.Collections.Immutable" Version="[1.7.1,)" />
     <PackageReference Include="System.Threading.Channels" Version="[6.0.0,)" />
+    <PackageReference Include="System.ValueTuple" Version="[4.5.0,)" Condition="'$(TargetFramework)' == 'net462'" />
```

Comment:
This is actually an interesting side-effect of bumping the MELA dependency.

Our original (ancient) MELA dependency resolves to [`2.0.2`](https://www.nuget.org/packages/Microsoft.Extensions.Logging.Abstractions/2.0.2#supportedframeworks-body-tab) which only includes a target for `netstandard2.0`, whereas starting with version [`6.0.0`](https://www.nuget.org/packages/Microsoft.Extensions.Logging.Abstractions/6.0.0#supportedframeworks-body-tab), MELA provided targets for `net461`, `netstandard2.0`, and `net6.0`. This continued into the `7.x` series with targets for `net462`, `netstandard2.0`, `net6.0`, and `net7.0`, as well as into the `8.x` series with targets for `net462`, `netstandard2.0`, `net6.0`, `net7.0`, and `net8.0`.

So by bumping to a modern MELA, our transitive reference to `System.ValueTuple` _via_ the `netstandard2.0` reference was lost, thus requiring us to take a direct dependency on `System.ValueTuple`.

Now it's been a minute since I had to deal with this in a real app, but when I last encountered this^ in the wild, IIRC, you always end up with a `System.ValueTuple` dependency _somewhere_ in the graph, so if you were skirting it before with a faux target of `net462 -> netstandard2.0`, you still needed the impl bits from `System.ValueTuple` available by run-time, else chaos ensued.